README.md

```diff
@@ -928,54 +928,57 @@ Similarly, when running tests in PyCharm, its own proprietary test discovery
+First, ensure that bzip2 and any other requirements for the Python build
+environment are met. See [pyenv wiki] for details:
 
 ```
 brew install openssl readline sqlite3 xz zlib bzip2
 ```
 
-Follow any additional steps that `brew` prompts for at the end of the installation. These  
-should include modifying path variables `LDFLAGS` and `CPPFLAGS`
-The commands from the `brew` output to modify the aforementioned path variables 
-can be placed in `~/.bash_profile` to make the change persistent
+Follow any additional steps that `brew` prompts for at the end of the
+installation. These should include modifying path variables `LDFLAGS` and
+`CPPFLAGS`. The commands from the `brew` output to modify the aforementioned
+path variables can be placed in `~/.bash_profile` to make the change persistent.
 
 Then install a patched Python 3.8.3 using `pyenv` by running:
 
 ```
 pyenv install --patch 3.8.3 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch)
 ```
 
-Users of macOS 11 or later may encounter `pip` installation errors due to `pip` not being able to 
-locate the appropriate wheels.
-The information below will help remedy this:
+Users of macOS 11 or later may encounter `pip` installation errors due to `pip`
+not being able to locate the appropriate wheels. The information below will
+help remedy this:
 
-[Resolution source](https://stackoverflow.com/a/63972598)
+[Resolution source]
 
-[macOS Big Sur Release Notes](https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_0_1-release-notes#Third-Party-Apps)
+[macOS 11 Release Notes]
 
-`pip` will not be able to locate the appropriate wheels due to the major release version of 
-Big Sur being incremented from `10.x` to `11.x`, instead pip will attempt to compile wheels 
-manually for wheels that it cannot locate. 
+`pip` will not be able to locate the appropriate wheels due to the major release
+version of macOS being incremented from `10.x` to `11.x`, instead pip will
+attempt to compile wheels manually for wheels that it cannot locate.
 
-In order to be able to run `make requirements` successfully, a backwards compatibility
-flag needs to be added to the `environment.local.py` file in the project root:
-
-```
-SYSTEM_VERSION_COMPAT=1
-```
+In order to be able to run `make requirements` successfully, a backwards
+compatibility flag needs to be added to the `environment.local.py` file in the
+project root. The flag is `SYSTEM_VERSION_COMPAT=1` and it needs to be inserted
+into the file (starting from line 25) as a key/value pair:
+`'SYSTEM_VERSION_COMPAT': 1`.
 
+[pyenv macOS 11 GitHub issue]:https://github.com/pyenv/pyenv/issues/1740
+[pyenv wiki]:https://github.com/pyenv/pyenv/wiki#suggested-build-environment
+[Resolution source]:https://stackoverflow.com/a/63972598
+[macOS 11 Release Notes]:https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_0_1-release-notes#Third-Party-Apps
```

Comment:
Defines these immediately after the paragraph containing the first reference. And only use reference style links where an inline link causes overly long lines within a paragraph or otherwise disturbs the flow.